mappings/net/minecraft/world/gen/feature/ColumnPlacer.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_4631 net/minecraft/world/gen/feature/ColumnPlacer
+	FIELD field_21227 minSize I
+	FIELD field_21228 extraSize I
```

Comment:
What's `extraSize`? Does it refer to `maxSize - minSize`? If yes, then I think `sizeRange` would be clearer. Also, what's the size? Is it the height of the column? Is it the radius/diameter?

Reply:
it is a column, so length. the data field is called size though

Reply:
> column, so length

Hm? A column is something that's vertical, so it has a height, not a length...

> the data field is called size though

OK then.